graph/propertyconventions.md

```diff
@@ -0,0 +1,70 @@
+# Guidelines for the conventions of properties on graph types
+
+## Updating read-only properties
```

Comment:
I was thinking to add this in a future PR. I don't know what we expect for creates, but I got a complete answer for updates, so I wanted to get it written down. 

If this is a blocker for you, I will work to understand the create behavior for this PR, otherwise I would like to update the document in the future for that use-case.